src/tests/ftest/pool/create_capacity.py

```diff
@@ -20,12 +22,284 @@ class PoolCreateCapacityTests(TestWithServers):
     :avocado: recursive
     """
 
+    def __init__(self, *args, **kwargs):
+        """Initialize a PoolCreateCapacityTests object."""
+        super().__init__(*args, **kwargs)
+
+        self.md_on_dev = ""
+        self.start_duration_timeout = 0
+        self.quantity = 0
+        self.meta_bytes = 0
+        self.data_bytes = 0
+
     def setUp(self):
         """Set up each test case."""
         # Create test-case-specific DAOS log files
         self.update_log_file_names()
         super().setUp()
 
+        self.md_on_dev = "md_on_scm"
+        if self.server_managers[0].manager.job.using_control_metadata:
+            self.md_on_dev = "md_on_ssd"
+
+        self.start_duration_timeout = self.params.get("start_duration", "/run/server_config/*", 0)
+        self.quantity = self.params.get("quantity", "/run/pool/*", 0)
+        self.meta_bytes = human_to_bytes(self.params.get("scm_size", "/run/pool/*", 0))
+        self.data_bytes = human_to_bytes(self.params.get("nvme_size", "/run/pool/*", 0))
+
+    def get_nsp_available_bytes(self, namespaces, namespace_id, cluster_bytes):
+        """Returns the available storage of a given NVMe namespace.
+
+        Args:
+            namespaces (dict): dmg json command output defining the NVMe namespaces.
+            namespace_id (int): Identifier of a namespace.
+            cluster_bytes (int): Size of an SPDK cluster.
+
+        Returns:
+            int: Available space of the given NVMe namespace.
+
+        """
+        for nsp in namespaces:
+            if nsp["id"] != namespace_id:
+                continue
+            return int(nsp["size"] / cluster_bytes) * cluster_bytes
+        self.fail(f"Namespace of id {namespace_id} could not be find")
```

Comment:
Only if you need to repush
```suggestion
        self.fail(f"Namespace of id {namespace_id} could not be found")
```

Reply:
- [ ] Fix spelling issue